apps/mull-api/src/app/event/event.service.ts

```diff
@@ -12,34 +12,34 @@ export class EventService {
     private eventRepository: Repository<Event>
   ) {}
 
-  findAll(): Promise<Event[]> {
+  events(): Promise<Event[]> {
     return this.eventRepository.find();
   }
 
-  findOne(id: number): Promise<Event> {
+  event(id: number): Promise<Event> {
     return this.eventRepository.findOne(id);
   }
 
-  findHostEvents(userId: number): Promise<Event[]> {
+  hostEvents(hostId: number): Promise<Event[]> {
```

Comment:
I was going to refactor this to one of the below. What ya think?
```suggestion
  getEventsHostedByUser(userId: number): Promise<Event[]> {
```
or
```suggestion
 eventsHostedBy(hostId: number): Promise<Event[]> {
```

Reply:
If we do change it to one of the above, we'd change `coHostEvents` and `participantEvents` as well below.

Reply:
I like the second one, more concise.

Reply:
I'll be putting your first suggestion as the service name, although the second is more concise it doesn't matter as much for the service method.

Reply:
For `participantEvents`, what do you think about `getEventsAttendedByUser`?

Reply:
I like it. We could also do `eventsAttendedByUser`

Reply:
Very clear to me